runtime/sema/check_event_declaration.go

```diff
@@ -61,6 +61,9 @@ func IsValidEventParameterType(t Type, results map[*Member]bool) bool {
 	case *AddressType:
 		return true
 
+	case *ReferenceType:
```

Comment:
How do references get serialized?

Reply:
Correct, it was possible to export the account types, but now the account "value" is a reference, so we should maybe keep support for allowing the use in an event.

I'll add some more tests around this.

Reply:
Added tests in cbbd45c70a471796b95002c33ea45da13adf670a